File: LogicadeProgramacao/Estrutura Sequencial/3. Python/SalarioMensal.py
valor_hora = float(input("Digite o valor que você ganha por hora: "))
horas_trabalhadas = float(input("Digite o número de horas trabalhadas no mês: "))

salario_bruto = valor_hora * horas_trabalhadas

desconto_ir = salario_bruto * 0.11
desconto_inss = salario_bruto * 0.08
desconto_sindicato = salario_bruto * 0.05

salario_liquido = salario_bruto - desconto_ir - desconto_inss - desconto_sindicato

print(f"Salário bruto: R$ {salario_bruto:.2f}")
print(f"Desconto do INSS: R$ {desconto_inss:.2f}")
print(f"Desconto do Sindicato: R$ {desconto_sindicato:.2f}")
print(f"Desconto do Imposto de Renda: R$ {desconto_ir:.2f}")
print(f"Salário líquido: R$ {salario_liquido:.2f}")

File: LogicadeProgramacao/Estrutura Sequencial/2. Java/SalarioMensal.java
import java.util.Scanner;

public class SalarioMensal {
    public static void main(String[] args) {
        double valorHora, horasTrabalhadas, salarioBruto, descontoIR, descontoINSS, descontoSindicato, salarioLiquido;
        Scanner sc = new Scanner(System.in);
        
        System.out.print("Digite o valor que você ganha por hora: ");
        valorHora = sc.nextDouble();
        
        System.out.print("Digite o número de horas trabalhadas no mês: ");
        horasTrabalhadas = sc.nextDouble();
        
        salarioBruto = valorHora * horasTrabalhadas;
        
        descontoIR = salarioBruto * 0.11;
        descontoINSS = salarioBruto * 0.08;
        descontoSindicato = salarioBruto * 0.05;
        
        salarioLiquido = salarioBruto - descontoIR - descontoINSS - descontoSindicato;
        
        System.out.printf("Salário bruto: R$ %.2f\n", salarioBruto);
        System.out.printf("Desconto do INSS: R$ %.2f\n", descontoINSS);
        System.out.printf("Desconto do Sindicato: R$ %.2f\n", descontoSindicato);
        System.out.printf("Salário líquido: R$ %.2f\n", salarioLiquido);
        
        sc.close();
    }
}
